tests/integration_tests/test_in_database_polling_autopilot_status.py

```diff
@@ -1,19 +1,24 @@
 import pytest
-from tests.integration_tests.utils.parameters import aws_params, setup_params
+from tests.integration_tests.utils.parameters import aws_params, \
+    reg_setup_params, cls_setup_params
 
 
 @pytest.mark.skipif(not aws_params.aws_access_key,
                     reason="AWS credentials are not set")
 def test_poll_sagemaker_autopilot_job_status(
         register_language_container, deploy_scripts, setup_database):
 
-    db_conn = setup_database
+    for setup_params in [reg_setup_params, cls_setup_params]:
```

Comment:
better use parameterized tests here, or have two test which call the run_test with different parameter, because with for loop you run two tests in on test and if the first fails the second doesn't run